backend/README.md

```diff
@@ -35,7 +34,8 @@ The `SECRET_KEY` should be sufficiently random. For example, you can use the fol
 
 Update the `DB_*` variables as needed, so they use the prepared PostgreSQL installation.
 
-4. Run `python manage.py makemigrations`
-5. Run `python manage.py migrate`
+3. Run `docker compose run django python manage.py makemigrations`
+4. Run `docker compose run django python manage.py migrate`
+4. Run `docker compose run django python manage.py collectstatic`
 
```

Comment:
I had the problem that the docker build was failing with the following Message `ERROR [internal] load metadata for docker.io/library/python:3`
The solution that worked for me was to remove the docker `config.json` with the command `rm  ~/.docker/config.json`. This should maybe be added here.